src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -59,8 +62,9 @@ public Controls(Subsystems s) {
 
 		launcherAmpPresetButton = codriveController.x();
 		launcherSubwooferPresetButton = codriveController.a();
+		launcherLowerPresetButton = codriveController.y();
 		// launcherPodiumPresetButton = codriveController.povLeft();
-		// launcherTrapPresetButton = codriveController.y();
+		launcherTrapPresetButton = codriveController.y();
```

Comment:
Note that this and the `launcherLowerPresetButton` are both bound to the same button.

Reply:
Temporary binding for testing, ill talk to drive team to figure out what the actual binding will be if/when it actually works

Reply:
fixed